eng/Versions.props

```diff
@@ -23,7 +23,7 @@
     <MicrosoftNetCompilersToolsetVersion>$(MicrosoftNETCoreCompilersPackageVersion)</MicrosoftNetCompilersToolsetVersion>
     <CodeStyleAnalyersVersion>$(MicrosoftNETCoreCompilersPackageVersion)</CodeStyleAnalyersVersion>
     <!-- Roslyn -->
-    <MicrosoftCodeAnalysisVersion>3.3.1</MicrosoftCodeAnalysisVersion>
+    <MicrosoftCodeAnalysisVersion>3.8.0-3.final</MicrosoftCodeAnalysisVersion>
```

Comment:
Please don't bump the repo version so aggressively. This prevents all analyzer packages to run on older bits.